doc/dirvish.txt

```diff
@@ -119,6 +119,10 @@ dirvish#add_icon_fn(fn)
         call dirvish#add_icon_fn({p -> p[-1:]=='/'?'📂':'📄'})
 <
 
+dirvish#apply_icons() 							       *dirvish#apply_icons()*
+	Reapply the icons returned by the functions registered with
+	|dirvish#add_icon_fn()|.
+
```

Comment:
this docs file doesn't use tabs, should this shouldn't either. i guess you have 'modeline' disabled?